.releaserc

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json.schemastore.org/semantic-release",
+  "branches": ["main", "chore-semantic-release"],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    [
+      "semantic-release-hackage",
+      {
+        "packageName": "hspec-golden",
+        "versionPrefix": "0."
```

Comment:
You can add `"publishDocumentation": true` if you want to create documentation with v2-haddock